tests/auto/baseline_collector.test.py

```
@@ -0,0 +1,454 @@
+        # Test with special characters in task type
+        result = self.collector.simulate_manual_task('task-with-special_chars')
+        assert result.task_type == 'task-with-special_chars'
+    
+    @patch('time.sleep')
+    def test_integration_with_scorer(self, mock_sleep):
+        """Test integration with PromptScorer"""
+        task_types = ['code_review', 'refactoring']
+        samples_per_type = 3
+        
+        # Collect baseline metrics
+        results = self.collector.collect_baseline_metrics(task_types, samples_per_type)
```

Ruff (0.11.9)

Local variable `results` is assigned to but never used

Remove assignment to unused variable `results`

(F841)